assembl/tests/fixtures/vote_session.py

```diff
@@ -29,13 +29,24 @@ def vote_session(request, test_session, discussion, timeline_vote_session,
     test_session.add(vote_session)
     test_session.flush()
 
+    identifier = 'voteSession{}'.format(vote_session.id)
+    short_title = u'Phase {}'.format(identifier)
+    root_thematic = Thematic(
+        discussion_id=discussion.id,
+        title=LangString.create(short_title, "en"),
+        identifier=identifier,
+        hidden=True)
+    discussion.root_idea.children.append(root_thematic)
+    test_session.flush()
+
```

Comment:
Why don't you use the create_root_thematic function here?

Reply:
there was a circular import issue

Reply:
Even if you put the import here locally?

Reply:
it's good, thanks